packages/bridge-schema/src/config/app.ts

```diff
@@ -254,6 +246,7 @@ export default defineUntypedSchema({
    * configuration. The name can refer to an indicator from [SpinKit](https://tobiasahlin.com/spinkit/)
    * or a path to an HTML template of the indicator source code (in this case, all the
    * other options will be passed to the template).
+   * @type {typeof import('../src/types/loading').NuxtOptionsLoadingIndicator | false}
```

Comment:
It's still missing the type `false` isn't it ?

Reply:
I have confirmed that setting `false` does not cause an error, what do you think is the problem?

Reply:
Here's the current issue I have
![image](https://github.com/nuxt/bridge/assets/75388869/727b7171-0e05-4b6c-b586-1840099e326a)

And here's the attribute in the `node_modules/@nuxt/bridge-schema/dist/index.d.ts` `ConfigSchema` interface
```ts
 /**
  * Used to set the default properties of the page transitions.
  * 
  * You can either pass a string (the transition name) or an object with properties to bind to the `<Transition>` component that will wrap your pages.
  * 
  * 
  * @see [vue@2 documentation](https://v2.vuejs.org/v2/guide/transitions.html)
  * 
  * @see [vue@3 documentation](https://vuejs.org/guide/built-ins/transition-group.html#enter-leave-transitions)
 */
 pageTransition: {
  [key: string]: any
 },
 ```